libs/components/toast/src/lib/modules/toast/toast.component.ts

```diff
@@ -112,12 +116,12 @@ export class SkyToastComponent implements OnInit, OnDestroy {
 
   private ngUnsubscribe = new Subject<void>();
 
-  private _toastType: SkyToastType;
-
   constructor(
     private changeDetector: ChangeDetectorRef,
     @Optional() private toasterService?: SkyToasterService
-  ) {}
+  ) {
+    this.updateIcon();
```

Comment:
I'd prefer to see this moved to `ngOnInit()` with the rest of the logic that initializes the state of the component, such as the `isOpen` state.

Reply:
Done